src/utils/deploy/hash-fns.mjs

```diff
@@ -120,14 +120,15 @@ const hashFns = async (
       displayName,
       generator,
       invocationMode,
+      buildData
     }),
   )
   const fnConfig = functionZips
-    .filter((func) => Boolean(func.displayName || func.generator || func.routes))
+    .filter((func) => Boolean(func.displayName || func.generator || func.routes || func.buildData))
     .reduce(
       (funcs, curr) => ({
         ...funcs,
-        [curr.name]: { display_name: curr.displayName, generator: curr.generator, routes: curr.routes },
+        [curr.name]: { display_name: curr.displayName, generator: curr.generator, routes: curr.routes, buildData: curr.buildData },
```

Comment:
```suggestion
        [curr.name]: { display_name: curr.displayName, generator: curr.generator, routes: curr.routes, build_data: curr.buildData },
```

I believe it needs to be snake-case:

https://github.com/netlify/bitballoon/blob/217df0749107d907b3dee20f9054cca6700d2a3e/app/controllers/api/v1/deploys_controller.rb#L99